fvtest/porttest/omrvmemTest.cpp

```diff
@@ -4207,6 +4207,24 @@ TEST(PortVmemTest, vmem_testGetProcessMemorySize)
 	EXPECT_EQ(0u, size) << "value updated when query invalid";
 }
 
+/**
+ * Sanity test of function to obtain available physical memory.
+ */
+TEST(PortVmemTest, omrvmem_get_available_physical_memory)
+{
+	OMRPORT_ACCESS_FROM_OMRPORT(portTestEnv->getPortLibrary());
+	uint64_t freePhysicalMemorySize = 0;
+	int32_t status = omrvmem_get_available_physical_memory(&freePhysicalMemorySize);
+	/* omrvmem_get_available_physical_memory is not supported on zos */
+#if !defined(J9ZOS390)
+	ASSERT_EQ(0, status) << "Non-zero status from omrvmem_get_available_physical_memory";
+	portTestEnv->log("freePhysicalMemorySize = %llu\n", freePhysicalMemorySize);
```

Comment:
While you're here, the printf string should be `...%" OMR_PRIu64 "...`, defined by `"omrformatconsts.h"`